src/PineDocs.php

```diff
@@ -139,5 +149,29 @@ static private function set_content_dir() {
 			self::$config->content_dir = xy_format_path(self::$config->content_dir, true);
 		}
 
+		static private function load_config_mathjax() {
+			$config_mathjax_path = '../config/mathjax.json';
+			if (!file_exists($config_mathjax_path)) {
+				// Create mathjax.json by copying mathjax-example.json.
+				$create = copy('../config/mathjax-example.json', '../config/mathjax.json');
+				if (!$create) {
+					exit('Error: Could not automatically create config/mathjax.json. You need to manually copy config/mathjax-example.json to config/mathjax.json');
+				}
+			}
+
+			// Read data
+			$data_mathjax = file_get_contents($config_mathjax_path);
+
+			if (!$data_mathjax) {
+				exit('Error: Could not read mathjax.json.');
+			}
+
+			// Check if json is ok
+			if (!json_decode($data_mathjax)) {
```

Comment:
Since there's no indication if the json has errors, it would be nice to throw an UI error to show the user that we used default mathjax configuration instead.
So call this here: `self::$errors[] = 'Invalid json in mathjax.json';`